cmd/info.go

```diff
@@ -99,6 +99,56 @@ func (o *infoOptions) run(cmd *cobra.Command) error {
 		if p.WorkspacesAvailable != nil {
 			displayInfoSliceItem(cmd, p.WorkspacesAvailable.Title, p.WorkspacesAvailable.Value)
 		}
+		if p.InteractiveSessionsCustomImageAllowed != nil {
+			displayInfoStringItem(cmd, p.InteractiveSessionsCustomImageAllowed.Title, &p.InteractiveSessionsCustomImageAllowed.Value)
+		}
+		if p.InteractiveSessionRecommendedJupyterImages != nil {
+			displayInfoSliceItem(cmd, p.InteractiveSessionRecommendedJupyterImages.Title, p.InteractiveSessionRecommendedJupyterImages.Value)
+		}
+		if p.SupportedWorkflowEngines != nil {
+			displayInfoSliceItem(cmd, p.SupportedWorkflowEngines.Title, p.SupportedWorkflowEngines.Value)
+		}
+		if p.CwlEngineTool != nil {
+			displayInfoStringItem(cmd, p.CwlEngineTool.Title, &p.CwlEngineTool.Value)
+		}
+		if p.CwlEngineVersion != nil {
+			displayInfoStringItem(cmd, p.CwlEngineVersion.Title, &p.CwlEngineVersion.Value)
+		}
+		if p.YadageEngineVersion != nil {
+			displayInfoStringItem(cmd, p.YadageEngineVersion.Title, &p.YadageEngineVersion.Value)
+		}
+		if p.YadageEngineAdageVersion != nil {
+			displayInfoStringItem(cmd, p.YadageEngineAdageVersion.Title, &p.YadageEngineAdageVersion.Value)
+		}
+		if p.YadageEnginePacktivityVersion != nil {
+			displayInfoStringItem(cmd, p.YadageEnginePacktivityVersion.Title, &p.YadageEnginePacktivityVersion.Value)
+		}
+		if p.SnakemakeEngineVersion != nil {
+			displayInfoStringItem(cmd, p.SnakemakeEngineVersion.Title, &p.SnakemakeEngineVersion.Value)
+		}
+		if p.DaskEnabled != nil {
+			displayInfoStringItem(cmd, p.DaskEnabled.Title, &p.DaskEnabled.Value)
+		}
+		if p.DaskEnabled != nil && strings.ToLower(p.DaskEnabled.Value) == "true" {
+			if p.DaskAutoscalerEnabled != nil {
+				displayInfoStringItem(cmd, p.DaskAutoscalerEnabled.Title, &p.DaskAutoscalerEnabled.Value)
+			}
+			if p.DaskClusterDefaultNumberOfWorkers != nil {
+				displayInfoStringItem(cmd, p.DaskClusterDefaultNumberOfWorkers.Title, &p.DaskClusterDefaultNumberOfWorkers.Value)
+			}
+			if p.DaskClusterMaxMemoryLimit != nil {
```

Comment:
You may want to amend the tests in order to have a case coverage for the new `info` endpoint replies?  Right new we have:

- testdata/inputs/info_big.json
- testdata/inputs/info_empty_inactivity_period.json
- testdata/inputs/info_small.json

You could introduce the workfow engine ones into "big" example, and you could introduce a new `info_dask.json` example for Dask variables, since the new Dask feature is optional...